src/redux/reducers/tripReducer.js

```diff
@@ -0,0 +1,147 @@
+import { filterSingleDimensionalArray } from '../../shared/filterArray';
+import { updateSingleDimensionalArray } from '../../shared/updateArray';
+import * as TripActions from '../actions/tripActions';
+
+const initialState = {
+  lastUpdated: null,
+  isFetching: false,
+  isSynced: false,
+  trips: []
+};
+
+export function tripReducer(state = initialState, action) {
+  switch (action.type) {
+    case TripActions.GET_TRIPS_FROM_DB_REQUEST:
+      return {
+        ...state,
+        lastUpdated: action.lastUpdated,
+        isFetching: action.isFetching,
+        isSynced: action.isSynced
+      };
+    case TripActions.GET_TRIPS_FROM_DB_FAILURE:
+      return {
+        ...state,
+        lastUpdated: action.lastUpdated,
+        isFetching: action.isFetching,
+        isSynced: action.isSynced
+      };
+    case TripActions.GET_TRIPS_FROM_DB_SUCCESS:
+      return {
+        ...state,
+        lastUpdated: action.lastUpdated,
+        isFetching: action.isFetching,
+        isSynced: action.isSynced,
+        trips: action.trips
```

Comment:
Should this be a separate action, like in other reducers? That is, an ADD_TRIPS_TO_STATE action? This works though.

Reply:
Whoops. Shouldn't be here.